int8 mode    # 0,global fram;   1,local fram
float64 pitch
float64 roll
float64 yaw
float64 time  # seconds
